metro.config.js

```diff
@@ -15,5 +15,13 @@ module.exports = {
     },
     resolver: {
         extraNodeModules: require('node-libs-react-native'),
+        sourceExts: [
```

Comment:
why?

Reply:
I used `.jsx` files to create a wrapper component for `Reanimated.View`, so they need to be included in the `sourceExts` in the metro config. An alternative would be, to use `getDefaultConfig()` from the `metro-config` package instead of listing all of the default extensions